src/country_info.rs

```diff
@@ -0,0 +1,171 @@
+use gettextrs::gettext;
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use once_cell::sync::Lazy;
+use std::collections::BTreeSet;
+use std::ops::Deref;
+use tdlib::types;
+
+#[derive(Clone, Debug, Default, glib::Boxed)]
+#[boxed_type(name = "BoxedCallingCodes")]
+pub(crate) struct BoxedCallingCodes(BTreeSet<String>);
```

Comment:
```suggestion
pub(crate) struct CallingCodes(BTreeSet<String>);
```

It's ok calling it just CallingCodes in this case. I usually use the Boxed* prefix only when I need to wrap a boxed type around an existing type.